spec/features/edit_album_spec.rb

```diff
@@ -58,8 +61,63 @@
     end
 
     scenario "User visits edit page" do
-      expect(page).to have_content @existing_album.name
+      visit edit_album_path(@existing_album)
+
+      expect(page).to have_selector("input[value='#{@existing_album.name}']")
       expect(page).to have_content @existing_album.description
+      expect(first('img')['src']).to have_content "ice-boats.jpg"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.